packages/components/src/primitives/svg/index.native.js

```diff
@@ -1,13 +1,26 @@
 import {
 	Path as PATH,
+	G as GROUP,
 	Svg,
 } from 'react-native-svg';
 
 export const Path = PATH;
+export const G = GROUP;
 export const SVG = ( props ) => {
-	return (
-		<Svg width={ props.width } height={ props.height } >
-			{ props.children }
-		</Svg>
-	);
+	if (props.width != undefined && props.height != undefined) {
+		return (
+			<Svg width={ props.width } height={ props.height } >
+				{ props.children }
+			</Svg>
+		);
+	} else {
+		// take viewport system to match the viewBox definition
+		// i.e. viewBox="0 0 24 24"
+		let viewBoxCoords = props.viewBox.split(' ');
+		return (
```

Comment:
That was the problem, RN complained about className being set. if omit takes it away it could work :).

Reply:
I think this is going to be a recurring issue, it would be nice to find a way to skip those `className` properties or convert them to `style`  which would work out of the box.

Reply:
tested it a bit and it seems to be working 👍 
addressed in 6d5b37db0